django/thunderstore/core/settings.py

```diff
@@ -673,3 +674,7 @@ def show_debug_toolbar(request: HttpRequest) -> bool:
     SECURE_SSL_REDIRECT = True
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
     SECURE_REDIRECT_EXEMPT = "/healthcheck/"
+
+# Token shared between Thunderstore apps to restrict access to OAuth
+# related endpoints.
+OAUTH_SHARED_SECRET = env.str("OAUTH_SHARED_SECRET")
```

Comment:
Should this default to something secure instead of an empty string? i.e. just random data, or have the API be disabled if no shared secret is defined?

Reply:
API view now checks the configuration as the first thing it does.